smith

```diff
@@ -19,8 +19,8 @@ search() {
 	if [[ -n "$results" ]]; then
 		while read -r url; do
 			head -1 "$url";
+			echo;
```

Comment:
It looks like you forgot to `git pull`. That echo statement should not be placed inside of the while loop.